net_ssl/_test.pony

```diff
@@ -640,7 +640,17 @@ primitive _TestSSLContext
         error
       end
 
-    let ssl_client = try sslctx.client()? else h.fail("failed getting ssl client session"); error end
-    let ssl_server = try sslctx.server()? else h.fail("failed getting ssl server session"); error end
+    let ssl_client = try
```

Comment:
The try block should start on the next line, indented one level

Reply:
is there any indentation of the `try` when it is on the next line?

Reply:
yes: https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#assignment